### PR TITLE
more Mob configstuff

### PR DESCRIPTION
**Pull request by Boy0000**

### CODE REVIEW

**Comment by Boy0000** [comment]
only the latest commit isnt merged in already. idk why it adds the others

File: servers/minecraft/plugins/Mobzy/mobs/passive/abyssal_snail.yml [hunk 1]
```diff
@@ -11,19 +11,21 @@ staticComponents:
...
-    maxExp: 25
+    minExp: 35
+    maxExp: 50
     drops:
       - item: 
           type: NAUTILUS_SHELL
           display-name: §x§7§8§5§F§3§4Abyssal Snail Shell
           lore: |
-            §e
+            §esomething
```
Comment (0ffz) [inline servers/minecraft/plugins/Mobzy/mobs/passive/abyssal_snail.yml:21]: Get these done!

File: servers/minecraft/plugins/Mobzy/mobs/passive/chimokami.yml [hunk 2]
```diff
@@ -7,14 +7,35 @@ staticComponents:
...
+          maxAmount: 2
+          dropChance: 0.7
 goals:
   1: !<minecraft:behavior.tempt>
     items: [CHICKEN, COOKED_CHICKEN]
   2: !<minecraft:behavior.avoid_player>
-    radius: 10
\ No newline at end of file
+    radius: 10
+  3: !<mobzy:target.attacker>
```
Comment (0ffz) [inline servers/minecraft/plugins/Mobzy/mobs/passive/chimokami.yml:39]: These should be under target selectors not goals. Also you would need a goal to follow and hurt the target

**Review by 0ffz — CHANGES_REQUESTED** [review]
Just by the way, if this isn't finished yet, I think it's better to make it a draft PR.

File: servers/minecraft/plugins/Mobzy/mobs/passive/abyssal_snail.yml (hunk 1 above)
Comment (Boy0000) [reply servers/minecraft/plugins/Mobzy/mobs/passive/abyssal_snail.yml:21]: its so kniho can fork the thing and rewrite it and not dm me stuff on discord for me to add

**Pull request merged**